src/actions/types.js

```diff
@@ -0,0 +1,2 @@
+const GET_ARTICLES = 'GET_ARTICLES';
+export default GET_ARTICLES;
```

Comment:
You shouldn't use export default here because this file is going to export a lot of type constants

Reply:
You haven't worked on this feedback